addons/survey/views/survey_survey_views.xml

```diff
@@ -166,11 +166,22 @@
             </form>
         </field>
     </record>
+    <record id="survey_sample_form" model="ir.ui.view">
```

Comment:
Guidelines: Note that xml id should be something like "survey_survey_view_form_sample" . And name "survey.survey.view.form.sample" .